shutorial-admin.sh

```diff
@@ -42,6 +44,7 @@ squashfs_build() {
       --customize-hook='chroot "$1" passwd --delete root' \
       --customize-hook='echo shutorial > "$1/etc/hostname"' \
       --customize-hook='echo "127.0.0.1 localhost host" > "$1/etc/hosts"' \
+      --mode="sudo"\
```

Comment:
That's the only part where I have a question on this very good patch set (thanks for that). Why did you had to go for the sudo mode? I prefer to not use sudo when I can avoid.

Reply:
I removed it because it's not needed anymore. I just added it because of rights problems that I encountered at some point.